### PR TITLE
chore: 🤖 Retire rose form textarea

### PR DESCRIPTION
**Pull request by NatashaKab**
✅ Closes: ICU-9425

:tickets: [Jira ticket](https://hashicorp.atlassian.net/browse/ICU-9425)

## Description
Retire rose form textarea

:rose: [Rose preview](https://boundary-ui-storybook-git-icu-9425-retire-rose-481df7-hashicorp.vercel.app/?path=/story/rose-form-textarea-removed--basic-textarea)

### CODE REVIEW

**Review by lisbet-alvarez — APPROVED** [review]
Amazing work! Looks great!

File: addons/rose/addon/components/rose/form/index.stories.mdx [hunk 1]
```diff
@@ -18,7 +18,10 @@ export const Template = (args) => ({
         as |form|
       >
         <form.input @label='Label' @helperText='Helper text' @value='Text' />
-        <form.textarea @label='Label' @helperText='Helper text' @value='Text' />
+        <Hds::Form::Textarea::Field @value="This is my description" as |F|>
```
Comment (DhariniJeeva) [inline addons/rose/addon/components/rose/form/index.stories.mdx:21]: This should include `disabled={{form.disabled}}`, same as the below select field to show the usage

File: addons/rose/addon/components/rose/form/index.stories.mdx [hunk 2]
```diff
@@ -78,8 +81,10 @@ export const Template = (args) => ({
   as |form|
 >
   <form.input @label='Label' @helperText='Helper text' @value='Text' />
-  <form.textarea @label='Label' @helperText='Helper text' @value='Text' />
-
+  <Hds::Form::Textarea::Field @value='This is my description' as |F|>
```
Comment (DhariniJeeva) [inline addons/rose/addon/components/rose/form/index.stories.mdx:84]: This should include disabled={{form.disabled}}, same as the below select field to show the usage

File: addons/rose/tests/integration/components/rose/form-test.js [hunk 3]
```diff
@@ -46,7 +46,10 @@ module('Integration | Component | rose/form', function (hooks) {
         as |form|
       >
         <form.input @label="Input" @name="input-field" />
-        <form.textarea @label="Textarea" @name="textarea-field" />
+        <Hds::Form::Textarea::Field @value="This is my description" name="textarea-field" as |F|>
```
Comment (DhariniJeeva) [inline addons/rose/tests/integration/components/rose/form-test.js:49]: This should include disabled={{form.disabled}}, same as the below select field to show the usage

File: addons/rose/addon/components/rose/form/textarea/index.stories.mdx [hunk 4]
```diff
@@ -1,39 +1,26 @@
...
-  @value='Descriptive content'
-  @label='Description'
-  @helperText='Enter a description'
-  @error={{false}}
-  @disabled={{false}}
-  readonly={{false}}
-  @link='www.example.net'
-  @linkText='link text goes here'
-/>
+<Hds::Form::Textarea::Field @value='This is my description' as |F|>
```
Comment (DhariniJeeva) [inline addons/rose/addon/components/rose/form/textarea/index.stories.mdx:12]: This should include disabled={{form.disabled}} to show the usage

**Review by DhariniJeeva — CHANGES_REQUESTED** [review]
can we include the usage of the `disabled` attribute in the textarea storybook similar to the select field?

**Review by calcaide — CHANGES_REQUESTED** [review]
+1 to @DhariniJeeva comments. Great job Natasha!! looking good 😃

**Review by calcaide — APPROVED** [review]
Let's goooo @NatashaKab !!! great job!! 💪

**Review by DhariniJeeva — APPROVED** [review]

**Review by ZedLi — APPROVED** [review]

**Pull request merged**